packages/affiliates/libs/synthPrices.js

```diff
@@ -0,0 +1,23 @@
+const { createReferencePriceFeedForEmp, Networker } = require("@uma/financial-templates-lib");
```

Comment:
this is the new price feed for developer mining. it returns the value of a synthetic, denominated in collateral currency, for a given `empAddress` between `from` and `to`.

Reply:
I really like that we're re-using the financial-templates-lib SDK here